packer/packer.json

```diff
@@ -47,6 +47,24 @@
         "kubernetes_version": "{{user `kubernetes_version`}}",
         "kubernetes_cni_version": "{{user `kubernetes_cni_version`}}"
       }
+    },
+    {
+      "name": "gcp-ubuntu-16.04",
+      "type": "googlecompute",
+      "project_id": "{{user `project_id`}}",
+      "source_image": "ubuntu-1604-xenial-v20180306",
+      "ssh_username": "packer",
+      "zone": "us-east1-b",
+      "metadata": {
+        "build_version": "{{user `build_version`}}",
+        "source_ami": "ubuntu-1604-xenial-v20180306",
```

Comment:
should this be named something other than "ami" ?

Reply:
yeah, probably